docs/rules/1020.md

```diff
@@ -2,7 +2,7 @@
 
 **Description**: Checks whether an existing enum property has added more allowed values from the previous specification. 
 
-**Cause**: This is considered a breaking change. This change requires new api-version.
+**Cause**: This is considered a breaking change in the same api-version but not breaking in new api-version .
```

Comment:
How to figure out the same api-version or cross api-version? Are these done by unified pipeline?

Reply:
Yes, it's done in the unified pipeline, fix this doc to avoid confusing PR authors and reviewers